### PR TITLE
Update for unionnorteamericana.miraheze.org

### PR DESCRIPTION
**Pull request by CnocBride**
Adding Graph and Labelled Section Transclusion per request T2147

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1417,6 +1417,7 @@
 		'bigforestwiki' => true,
 		'calexitwiki' => true,
 		'test1wiki' => true,
+		'unionnorteamericanawiki' = true,
```
Comment (Reception123) [inline LocalSettings.php]: please use => not = in this case.

File: LocalSettings.php [hunk 2]
```diff
@@ -1496,6 +1497,7 @@
 		'test1wiki' => true,
 		'sidemwiki' => true,
 		'testwiki' => true,
+		'unionnorteamericanawiki' = true,
```
Comment (Reception123) [inline LocalSettings.php]: same as above.

Comment (CnocBride) [reply LocalSettings.php]: Crap, I must have forgotten to enter it in.

Comment (Reception123) [reply LocalSettings.php]: still missign the second one "=>"

**Comment by CnocBride** [comment]
Now if this isn't correct I dunno what will xD.

**Pull request merged**